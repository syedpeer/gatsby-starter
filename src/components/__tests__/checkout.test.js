import Checkout  from '../checkout'
import React from 'react'
import { render } from 'react-testing-library'
import renderer from 'react-test-renderer'

describe(`Checkout`, () =>
	it.skip(`renders correctly with react-testing-library`, () => {
		const { getByTestId } = render(<Checkout />)
		expect(getByTestId(`checkout-button`)).toHaveTextContent(`BUY NOW`)
	}),
	it(`renders correctly with react-test-renderer`, () => {
		const tree = renderer.create(<Checkout />).toJSON()
		expect(tree).toMatchSnapshot()
	}),
)
