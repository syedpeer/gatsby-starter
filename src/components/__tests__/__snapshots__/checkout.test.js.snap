// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with react-test-renderer 1`] = `
<div>
  <h4>
    Spend your Money!
  </h4>
  <p>
    Use any email, 4242 4242 4242 4242 as the credit card nmumber, any 3 digit number, and any future date of expiration.
  </p>
  <button
    data-testid="checkout-button"
    disabled={false}
    onClick={[Function]}
  >
    BUY NOW
  </button>
  
</div>
`;
